### ISSUE TITLE
Expand mdloader to support nvm usage 

### ISSUE THREAD

**Issue by drashna** [Massdrop/mdloader#14]
Specifically, so this PR can be supported, so we can have persistent settings on the ATSAM boards.

https://github.com/qmk/qmk_firmware/pull/6068

**Comment by daltona** [comment]
https://github.com/daltona/mdloader/tree/enable-smart-eeprom while still Work In Progress, adds an option that will update the NVM user row in order to enable the smart eeprom that is used in the mentioned qmk_firmware.
For those interested to test.

**Comment by drashna** [comment]
Also, figured I should tag @patrickmt on this.

**Comment by robhaswell** [comment]
Hi, what's blocking getting this branch merged? If it is real-world testing, can I help? As a user, I think persistent settings should be landed for the Drop boards ASAP.

**Comment by aoswalt** [comment]
Would love to get an update on this.

**Comment by rshutt** [comment]
I agree.  I really rather not have to maintain a fork of this functionality.  I'm guessing others already have?
